assets/css/header.css

```diff
@@ -165,7 +165,7 @@ a:active {
 
 .banner-link {
   padding: 15px;
-  font-size: 1rem;
+  font-size: 14px !important;
```

Comment:
Instead of changing it to px can you add media queries instead? When the screen is somewhat larger the buttons look very small

Reply:
while setting it to `font-size: 1rem;` it is overflowing dark area.
Actually working on it latter on i thought of a new way, if the shades height is changed it can be achieved.